AppDashboard/lib/app_dashboard_helper.py

```diff
@@ -319,6 +319,37 @@ def get_app_port(self, appname):
         "that it runs on.".format(appname))
 
 
+  def get_ssl_port(self, appname):
+    """ Queries the UserAppServer to learn the https port the named application.
```

Comment:
Please change "learn the https port the named application." to "learn the https port of the named application.".